README.md

```diff
@@ -8,6 +8,12 @@ Free software projects that maintain a P2P network aim for diversity of peers to
 
 Kartograf is a free software project that generates such map files, for the start IP to AS maps, in a trust-minimized and transparent way. It pulls data from different public sources and combines them in a purely functional way to minimize the possibility of accidental BGP leaks or malicious hijacks ending up in the resulting map. Additionally, the software provides informative, easy-to-parse logging and preserves intermittent artifacts. Projects utilizing Kartograf will then publish this information which allows their users to audit the generation process of the map and verify the integrity of the shipped result file by repeating the process, similar to reproducible build systems.
 
+The following data sources can be used, and when enabled, will be used in decreasing order of priority:
+
+- [RPKI](https://www.ripe.net/manage-ips-and-asns/resource-management/rpki/what-is-rpki/)
+- [RIRs IRR](https://www.arin.net/resources/manage/irr/)
```

Comment:
I think this is a bit more understandable and helps the people that need to look up what these letters mean :) (IRR and RIR are such generic abbreviations that I don't think they are necessarily the first thing that shows up in most online search venues)

```suggestion
- [IRR DBs of all five RIRs](https://www.arin.net/resources/manage/irr/)
```